zeppelindemo/resources/projects.xml

```diff
@@ -54,16 +54,19 @@ documents with SQL, Scala and virtually any other language or data-processing-ba
 $ ssh kaa@YOUR-SANDBOX-IP
 password: kaa
 </pre>
-3. Download Apache Zeppelin from the <a href="http://zeppelin.incubator.apache.org/download.html">official page</a> and install it to your Sandbox. Execute
-following commands:
+3. Download Apache Zeppelin and install it to your Sandbox.
+Execute following commands:
 <pre>
 $ cd /opt
-$ sudo wget http://www.eu.apache.org/dist/incubator/zeppelin/0.5.6-incubating/zeppelin-0.5.6-incubating-bin-all.tgz
-$ sudo tar -xzvf zeppelin-0.5.6-incubating-bin-all.tgz
-$ sudo ln -s /opt/zeppelin-0.5.6-incubating-bin-all /opt/zeppelin
+$ sudo wget http://archive.apache.org/dist/zeppelin/zeppelin-0.6.1/zeppelin-0.6.1-bin-all.tgz
+$ sudo tar -xzvf zeppelin-0.6.1-bin-all.tgz
+$ sudo ln -s /opt/zeppelin-0.6.1-bin-all.tgz /opt/zeppelin
```

Comment:
No ".tgz" needed. This is mistake in ticket, I have correct it.
**sudo ln -s /opt/zeppelin-0.6.1-bin-all /opt/zeppelin**